migrations/20220219215644_create_books_favorites.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
```

Comment:
This function to create both tables looks so much better!

Reply:
Yeah I've learned so much from all the errors we got lol